doc/databaseinit.sql

```diff
@@ -0,0 +1,68 @@
+CREATE TABLE careprovider  (
```

Comment:
Should add some very simple sql comments noting what these do

Reply:
Done.